### PR TITLE
Add draft

### PR DESCRIPTION
**Pull request by hillaryfraley**

### CODE REVIEW

File: docs/blog/newsletter-may-2017.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
+.. post:: May 1, 2017
+   :tags: newsletter, 
+
+**************************************
+Write the Docs Newsletter - May 2017
+**************************************
+
+Write the Docs Portland in Less than 2 Weeks!
+#############################################
```
Comment (kelly-o) [inline docs/blog/newsletter-may-2017.rst]: missing an 'is' here, i.e. Write the Docs Portland is in Less than 2 Weeks!

Comment (kelly-o) [reply docs/blog/newsletter-may-2017.rst]: (also, aaaaaaaahhhhhhh 😱 )

File: docs/blog/newsletter-may-2017.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+The Challenges of Documentation in an Agile Environment
+#######################################################
+
+How can writers work with Agile dev teams? It's a perennial question that arose again this month and produced some excellent discussion. TL;DR: there are many models, and writers have to figure out what works best for their particular teams, processes, and products. Here's a round-up of the approaches that the WtD community came up with:
+
+* With teams that perform regression testing and user acceptance testing (UAT) after a scheduled sprint, docs are sometimes developed during these post-sprint cycles. In this kind of workflow, docs and testing are often closely allied.
+
+* Some writers are embedded on Agile dev teams, attend standups, and work on the regular sprint cycle, with stories assigned for doc; others work with parallel sprints, and devs/PMs serve as stakeholders.
+
+* Some of the "embedded" writers document features at the start of development, but doc is not released until feature complete. Other writers find that features change too much during dev cycles for this approach to be worthwhile.
```
Comment (kelly-o) [inline docs/blog/newsletter-may-2017.rst]: let's flesh out the second half of this first sentence: '...but the doc is not released until the feature is complete.'

File: docs/blog/newsletter-may-2017.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* `Linkurious <https://linkurio.us>`_
+* `Neo4j <https://neo4j.com/>`_
+* ...and good old `Visio <https://products.office.com/en-us/visio/flowchart-software>`_
+
+------------------------------------------
+
+Peer-to-Peer Teaching
+#####################
+
+When you're tasked with leading a workshop that will help your non-tech-writing colleagues write in clear, plain English, how can you help without embarrassing or alienating anyone? The discussion produced many thoughtful suggestions, including having the group dissect your own writing, but one idea stood out: peer-to-peer teaching using a whiteboard carousel. Here's how it works:
```
Comment (kelly-o) [inline docs/blog/newsletter-may-2017.rst]: the current placement of the 'your' at the end of this graf is a little unclear... maybe we could use this instead: 
'...including offering up some of your own writing for the group to dissect'?

File: docs/blog/newsletter-may-2017.rst (hunk 1 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: hahahah

File: docs/blog/newsletter-may-2017.rst (hunk 2 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: Got it

File: docs/blog/newsletter-may-2017.rst (hunk 3 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: Got it!

File: docs/blog/newsletter-may-2017.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+**************************************
+Write the Docs Newsletter - May 2017
+**************************************
+
+Write the Docs Portland is in Less than 2 Weeks!
+################################################
+
+`Write the Docs Portland <http://www.writethedocs.org/conf/na/2017/>`_ is almost here! If you're coming, we're looking forward to welcoming you. If you can't make it, maybe `Write the Docs Prague <http://www.writethedocs.org/conf/eu/2017/>`_ in September? Just a thought...
+
+The Portland conference is sold out again this year, but if you want to be on the waiting list in case of cancellations, please [**need to add this info**]
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2017.rst]: https://ti.to/writethedocs/write-the-docs-na-2017/ waiting list on a ticket type basis. There are approx 10 people on there now. 

Please rewrite as needed

File: docs/blog/newsletter-may-2017.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+Now, on to this month's newsletter, packed with gems we collected from the WtD Slack channel throughout the last 30-ish days.
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2017.rst]: WTD throughout

File: docs/blog/newsletter-may-2017.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+This month in WtD Slack, the question arose of how best to handle replaceable content in code snippets – in other words, any situation where we're prompting users to replace 'username' and 'password' with, y'know, their username and password. As usual, the community had a plethora of creative solutions to suggest.
+
+Some approaches were pretty lightweight and focused on adding clarity without introducing any coding behind the scenes:
+
+* Choose a formatting convention for any example data, so the replaceable parts of the snippet are easy to pick out visually.
+* Build up a small cast of fictional characters that you can write about throughout your docs. This lets you use a realistic name and password combo that your readers will have some context for.
+* Expand on the basic 'username' and 'password' with more expressive strings like, 'example_username', 'your_username', 'replace_with_username', and so on
+
+If you're looking for more dynamic solutions to the same problem, we've got some good options for you on that front as well:
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2017.rst:31]: Switching from 'the community had a plethora' to 'we've got'

File: docs/blog/newsletter-may-2017.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+* Automate the formatting for replaceable content by defining your own classes. DocBook, for example, will let you do this with the <replaceable> tag, and for Markdown, the same effect can be achieved with some help from `hercule <https://github.com/jamesramsay/hercule>`_ and bash scripts.
+* If you're using fictional characters, you can take it a step further and create actual accounts for them in your system. Then, whenever your real-life users try to execute something with a fictional account's data, you can throw an error message to prompt them to swap in their actual details.
+* For maximum coolness, you could build in a prompt at the top of your docs asking for the relevant details, then use that info to generate personalized code samples that are ready to go, right out of the gate. With this option, there are some authentication and security concerns to address, but `Stripe's API docs <https://stripe.com/docs>`_ are a great example of how this might look.
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2017.rst]: > maximum coolness 
grates on me

File: docs/blog/newsletter-may-2017.rst (hunk 4 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: added!

File: docs/blog/newsletter-may-2017.rst (hunk 5 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: I think I got them all

File: docs/blog/newsletter-may-2017.rst (hunk 6 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst:31]: Updated to "As usual, the discussion resulted in many creative solutions."

File: docs/blog/newsletter-may-2017.rst (hunk 7 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-may-2017.rst]: Added "grates on me"
j/k, I revised it to eliminate that phrase :) Thank you!

**Comment by plaindocs** [comment]
Looks good to me,

**Pull request merged**